model/src/payouts.rs

```diff
@@ -106,6 +108,50 @@ impl Payouts {
         })
     }
 
+    pub fn new_quanto(
+        (position, role): (Position, Role),
+        initial_price: u64,
+        n_contracts: u64,
+        (leverage_long, leverage_short): (Leverage, Leverage),
+        n_payouts: usize,
```

Comment:
Do you think we can make `n_payouts` dynamic depending on a `step` value? I mean: What I'd like to see is payouts in e.g. `$10` increments (or ideally in `10` sats increments).

Reply:
We absolutely can replace `n_payouts` with a `max_interval_size` argument. I just wanted to keep it in line with the other API.

Mind you that I think the reason why we use `n_payouts` is so that we can more or less consistently control the number of CETs per CFD.

Reply:
I've created https://github.com/itchysats/itchysats/issues/2632.